src/RoslynPad.Roslyn/CodeFixes/CodeFixService.cs

```diff
@@ -355,6 +356,33 @@ await diagnosticsWithSameSpan.OrderByDescending(d => d.Severity)
                 result.Add(codeFix);
             }
 
+            private FixAllState CreateFixAllState(
```

Comment:
Did you just take the fix all provider? Perhaps it's a good idea to copy the whole `CodeFixService` class again.

Reply:
I wan't aware this was a copy of a Roslyn class but now I've looked I made the same changes that were made in the commit as the was made to Roslyn when they moved the FixAllDiagnosticProvider class. Comparing the two there are some differences with the Roslyn version but I'm not 100% sure what should be merged. I assume you've made some changes to it after copying it, or is it just a straight copy that you have updated every so often?

Reply:
@aelij Hi, did you see my comment above?

Reply:
I did make some minor changes, I just don't remember exactly what. Let's leave it as-is for now. Did you check that code fixes work?

Reply:
Yes, at the moment we're using these changes a part of our project as we need the 2.6.1 support. The code is also a straight copy from Roslyn.